evap/results/templates/results_semester_detail.html

```diff
@@ -38,13 +38,23 @@ <h3>{{ semester.name }}</h3>
                                 {% if course|can_user_see_results:request.user %}data-url="{% url 'results:course_detail' semester.id course.id %}"{% endif %}>
                                     <td data-order="{{ course.name }}">
                                         {% if course.state == 'in_evaluation' %}
-                                            <span data-toggle="tooltip" data-placement="top" class="fas fa-play" title="{% trans 'This course is still in evaluation.' %}"></span>
+                                            <span data-toggle="tooltip" data-placement="top" class="fas fa-play" title="{% trans 'In evaluation' %}"></span>
                                         {% endif %}
-                                        {% if course.state == 'evaluated' or course.state == 'reviewed' %}
-                                            <span data-toggle="tooltip" data-placement="top" class="fas fa-pause" title="{% trans 'This course has not been published yet.' %}"></span>
+                                        {% if course.state == 'evaluated' %}
+                                            <span data-toggle="tooltip" data-placement="top" class="fas fa-chart-bar icon-yellow" title="{% trans 'Results not yet published' %}"></span>
+                                        {% endif %}
+                                        {% if course.state == 'reviewed' %}
+                                            {% if course.is_single_result or course.final_grade_documents or course.gets_no_grade_documents or not course.is_graded %}
+                                                <span data-toggle="tooltip" data-placement="top" class="fas fa-chart-bar icon-red" title="{% trans 'Results not yet published although they probably could be' %}"></span>
+                                            {% else %}
+                                                <span data-toggle="tooltip" data-placement="top" class="fas fa-chart-bar icon-yellow" title="{% trans 'Results not yet published' %}"></span>
+                                            {% endif %}
```

Comment:
isn't there a lot more of the icons from the staff semester view that could be put in here? either put all of that in here or the bare minimum, i would say.